magento/magento2
# Fixed issue Magento Community Edition: Version 2.4.5 - Date Picker design getting disturb

### Description (*)

### Related Pull Requests


### Fixed Issues (if relevant)

1. Fixes #36499

### Manual testing scenarios (*)
1. Magento Community Edition 2.4.5 without sample data is installed.
2. Login into admin > Marketing > Cart Price Rules
3. Click on date picker of start column in grid, Select the date from date picker
4. After selecting date from the date picker, calendar icon should be visible over there.

### Questions or comments


### Contribution checklist (*)
